platform_api/user.py

```diff
@@ -9,10 +9,17 @@
 logger = logging.getLogger(__name__)
 
 
-def make_job_uri(user: AuthUser, cluster_name: str, org_name: Optional[str]) -> URL:
-    if org_name is None:
-        return URL.build(scheme="job", host=cluster_name) / user.name
-    return URL.build(scheme="job", host=cluster_name) / org_name / user.name
+def make_job_uri(
```

Comment:
we use it only in `infer_permissions_from_container`, maybe let's move it there?

Reply:
done.

Reply:
in a follow up PR.